src/index.html

```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
Don't add blank lines between parent and child tags, remove them everywhere
```suggestion
<html lang="en">
  <head>
```